frontend/src/components/Documentation.vue

```diff
@@ -156,12 +156,21 @@
             <span class="icon pr-5 is-large has-text-info"><i class="fa fa-connectdevelop"></i></span>
             Interaction Partners
           </h4>
-          <p>For a given metabolite or gene, this tool renders a connected graph of the other metabolites and genes with which it interacts via shared reaction(s). The metabolite or gene of interest is centered on the graph. Connectivity is determined based on the reactions with which this metabolite or gene is associated, i.e. an edge between two nodes may represente a connections that occurs in multiple reactions. For medium-sized networks of interaction partners, there is a prompt before generating the graph. For very large networks (e.g. for H<sub>2</sub>O) the graph will not be generated. One can left-click a node to display a context menu with 3 options:</p>
-          <ul>
-            <li>Load interaction partners: reload the interaction partners graph with the clicked node as the new central node.</li>
-            <li>Expand interaction partners: add additional interaction partner nodes for the clicked node to the graph. Expanded interactions are represented with dashed lines.</li>
-            <li>Highlight reaction: some nodes may be involve in many different reactions. Select a reaction from the list to show the other interaction partners associated with the selected reaction (other interaction partners will be grayed-out). The directionality of each edge is indicated as a triangle, or diamonds in case of reversible reactions. To remove the highlight, click on the <span><i class="fa fa-eraser"></i></span> eraser button at the top of the graph.</li>
-          </ul>
+          <div id="interaction-partners-intro">
+            <video id="interaction-partners-video"
+                   poster="@/assets/interPart-cover.jpg" playsinline controls muted loop>
+              <source src="@/assets/interPart.mp4" type="video/mp4">
```

Comment:
It will be better to add a caption for the demo video and mention it also in the text.

Reply:
Thanks! I added a caption. I think the current text already accompanies the video quite well but if you have any thoughts on where it would be appropriate to add the mention please feel free to add a commit.